### ISSUE TITLE
2.6 - Background color on code ellipsis is too white

### ISSUE THREAD

**Issue by mtancoigne** [cquanu/github-dark#39]
I may change this myself but don't want to mess with pure css... (that would be great to use something like Sass or Less to generate it...)

#### What's the issue:
Background color on code ellipsis is too white
 
 #151f29
#### Steps to reproduce:
 - Search for  something in a repo (i.e. [this](https://github.com/CakeDC/users/search?utf8=%E2%9C%93&q=remember)):
 - Results will be displayed with code ellipsis

##### Current:
![selection_026](https://cloud.githubusercontent.com/assets/1732268/21845889/ce3950d4-d7f4-11e6-89e8-4b1641e4472f.png)

##### Proposition:
![selection_025](https://cloud.githubusercontent.com/assets/1732268/21845901/d4bc77d8-d7f4-11e6-96cf-892942bb4279.png)

#### Solutions:
Change the background to `#151f29`:

```css
.code-list .divider .blob-num, .code-list .divider .blob-code{
  background-color:#151f29;
}
```

#### What version is affected by this problem:
Change for version 2.6

**Comment by cquanu** [comment]
This will be fixed in the next update. Thanks @mtancoigne !

**Comment by mtancoigne** [comment]
You're welcome (I updated the proposition as the CSS selector was wrong)

**Comment by cquanu** [comment]
Noted. Thanks

**Comment by tgy** [comment]
Has this been fixed?

**Comment by cquanu** [comment]
Fixed in #43
